### PR TITLE
Fix progress bar drawing and add new spritesheet

### PR DESCRIPTION
**Pull request by steiner26**
## Overview

This PR fixes and improves the drawing of opponent progress bars. It also includes a new spritesheet for the animation.

## Changes Made

* Fixed drawing of opponent progress bars (I had the directions of the reflection mixed up 😔)
* Added new spritesheet for opponent progress bar animation
* Made it so the progress bar is always drawn at full scale but moves in and out based on the opponent's progress
* Made it so the bar slides in and out smoothly instead of jumping to its new position

## Screenshots

<details>

  <summary>Normal animation and growth</summary>

  

Animation and Growth

</details>

<details>

  <summary>Hit animation</summary>

  

Hit Animation

</details>

### CODE REVIEW

**Comment by henryz2000** [comment]
how does the hit animation feel from a game feel perspective?

File: cugl/build-apple/Box2D.xcodeproj/xcuserdata/brandon.xcuserdatad/xcschemes/xcschememanagement.plist [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (wlong123) [inline cugl/build-apple/Box2D.xcodeproj/xcuserdata/brandon.xcuserdatad/xcschemes/xcschememanagement.plist:1]: should this be included in the pr?

**Review by wlong123 — APPROVED** [review]

**Comment by wlong123** [comment]
> how does the hit animation feel from a game feel perspective?

plus 1 to this. the screen recording for hitting a player looks a bit awkward

**Comment by steiner26** [comment]
I think the game feel is pretty good - you first notice that a bunch of stardust has spawned and then you can look around to see whose progress bar is not glowing as much

**Pull request merged**